### ISSUE TITLE
Input Group: Date selector is not keyboard accessible

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#3412]
**Describe the issue. What is the expected and unexpected behavior?**
The dropdown date selector is not accessible by keyboard. If it is clicked by mouse, it is then able to be interacted with, but currently there is no way of getting there for a keyboard user.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
This is a bug.

**What is your product and what release version are you targeting?**

**Comment by jessiehuff** [comment]
After looking into this issue, it seems that this is browser native, and there is alternative functionality for keyboard users which is appropriate based on discussion.
